src/Orchard.Recipes.Abstractions/Services/IRecipeExecutor.cs

```diff
@@ -1,10 +1,10 @@
-﻿using Orchard.Recipes.Models;
-using System.Threading.Tasks;
+﻿using System.Threading.Tasks;
+using Orchard.Recipes.Models;
 
 namespace Orchard.Recipes.Services
 {
     public interface IRecipeExecutor
     {
-        Task<string> ExecuteAsync(string executionId, RecipeDescriptor recipeDescriptor);
+        Task<string> ExecuteAsync(string executionId, RecipeDescriptor recipeDescriptor, object environment);
```

Comment:
The goal of this object is to contain any value that a recipe might require. Recipes will be able to expose a `parameters` object describing the list of values they require, so we can build some UI and gather the value for it. What type of object would be better? I could have chosen a Dictionary, a dynamic, a JObject, in the end I decided on object. Make sense?